src/main/index.js

```diff
@@ -68,6 +68,8 @@ function init () {
   app.on('ready', function () {
     isReady = true
 
+    // require('electron').BrowserWindow.addDevToolsExtension('/Users/feross/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/0.15.3_0/')
```

Comment:
probably React DevTools

On 23 Aug 2016 06:18, "Benjamin Tan" <notifications@github.com> wrote:

> In src/main/index.js
> <https://github.com/feross/webtorrent-desktop/pull/806#discussion_r75862892>
> :
>
> > @@ -68,6 +68,8 @@ function init () {
> >    app.on('ready', function () {
> >      isReady = true
> >
> > +    // require('electron').BrowserWindow.addDevToolsExtension('/Users/feross/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/0.15.3_0/')
>
> What's this? Interested to know :)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/feross/webtorrent-desktop/pull/806/files/509691a85a86b51f004b8b5cbcdfe3ea39cecd49..157226f75bbb2397a43ebc11d266fbaea3ef3b3f#r75862892>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAKVQKoLwC89gQJZMCLgr__c4Pe0fF7dks5qivMqgaJpZM4Jqc9q>
> .
>